### PR TITLE
the lord has spoken

### PR DESCRIPTION
**Pull request by lord-travis**
hurry theyre listening to me as i type this out the van is outside my house they are using 1ghz radio antennas pointed at the bugs in the wall to illuminate the microphones and listen to my conversation the roaches in the floorboards are watching me as i sleep help me the vaccine contains microchips that run the windows kernel with the smb bug that caused wannycry to spread the vaccine is ransomeware theyre holding the us dollar hostage with the stimulus checks buy etherium buy etherium

### CODE REVIEW

**Comment by lord-travis** [comment]
do it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itdo it do itvv do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her do it for her

**Comment by lord-travis** [comment]
no problem boss ill have it done right away sir

**Comment by lord-travis** [comment]
good man good im looking forward to it sir


.

**Comment by travisjayday** [comment]
aight man thank for review im gonna go now

File: pay.txt [hunk 1]
```diff
@@ -1,46 +1,46 @@
 total payment w/o deposit  = 8814.02
-total payment w/o deposit / person = 2203.505
-assuming deposit 		   = 500 
+total payment w/o deposit / person = 2203.50
```
Comment (travisjayday) [inline pay.txt:2]: what is this fam. the total price is 2203.505!!!! NOT 2203.50!!! Are u retarded or wut m8

Comment (travisjayday) [reply pay.txt:2]: @lord-travis

**Review by travisjayday — COMMENTED** [review]
NONONO

File: pay.txt (hunk 1 above)
Comment (lord-travis) [reply pay.txt:2]: no WRONG PLEASE DONT HIT ME PLEASE IM SORRY

**Review by travisjayday — CHANGES_REQUESTED** [review]
bad bad code

**Pull request closed without merging**